src/main/java/com/cglx/Calculator.java

```diff
@@ -31,7 +31,14 @@ public int square(int a)
         return a * a;
     }
 
-    public boolean areEqual(float f1, float f2) {
-        return f1==f2;
+    private void readFile() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "readFile" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-01.ahprod.adhaven.com:9990/coding_rules#rule_key=squid%3AUnusedPrivateMethod)